### PR TITLE
Update to STATICFILES_DIRS and STATIC_ROOT

### PR DESCRIPTION
**Pull request by ll4407**
- I renamed the current 'static' directory to 'static_dev'
- I updated STATICFILES_DIRS to point to 'static_dev'
- I defined STATIC_ROOT as pointing to a directory in the root folder called 'static'

Rationale:  I had trouble running 'python manage.py collectstatic' during deployment.  The STATIC_ROOT was missing from settings.py.  I then tried commenting out STATICFILES_DIR.  This allowed me to run the 'collectstatic' command, but when I ran the application locally, the background image and styles.css file would not load.  After running the solution described below, I was able to run the 'collecstatic' command as well as have the background image and styles.css file load during development. 

Development:

When you're in a development environment and just want Django to serve static files, you'd typically use STATICFILES_DIRS to point to the locations of static files:

settings.py:

STATIC_URL = '/static/'
STATICFILES_DIRS = [os.path.join(BASE_DIR, "static_dev")]  # Note the change from "static" to "static_dev" or any other name you prefer.

Django will automatically find and serve static files from the static_dev directory during development. When you create a new app or place static files in the root, you'd put them in this static_dev directory.

Deployment:

For deployment, we use collectstatic to gather all the static files and place them in a directory suitable for serving by a web server or service like AWS Elastic Beanstalk:

In settings.py:

STATIC_URL = '/static/'
STATICFILES_DIRS = [os.path.join(BASE_DIR, "static_dev")]
STATIC_ROOT = os.path.join(BASE_DIR, "static")

Before deploying, we run:

'python manage.py collectstatic'

This command will take all the static files from your apps and from STATICFILES_DIRS and place them in the STATIC_ROOT (static directory in this case). Now, Elastic Beanstalk or any other service can serve these files from the static directory.

### CODE REVIEW

File: static_dev/css/styles.css
Comment (ll4407) [inline static_dev/css/styles.css:1]: moved to static_dev/css/styles.css

File: static_dev/roleplaydate/images/background.png
Comment (ll4407) [inline static_dev/roleplaydate/images/background.png:1]: Moved to static_dev folder

File: roleplaydate/settings.py
Comment (ll4407) [inline roleplaydate/settings.py:1]: Uncommented STATICFILES_DIRS.  STATICFILES_DIRS now points to 'static_dev' folder instead of 'static'

**Review by aya0221 — APPROVED** [review]
Leu, LGTM! Thanks for applying the best practice!

**Pull request merged**